ipn/ipnlocal/local.go

```diff
@@ -6023,3 +6025,138 @@ func mayDeref[T any](p *T) (v T) {
 	}
 	return *p
 }
+
+var errNoExitNodes = errors.New("no exit nodes available")
+var errUnableToPick = errors.New("unable to pick candidate")
+
+// SuggestDERPExitNode returns a tailcfg.StableNodeID of a suggested exit node given the local backend's netmap and last report.
```

Comment:
This comment seems to be out of date. And I don't understand what DERP has to do with exit nodes?